packages/nx/src/schematics/application/files_angular/package.json

```diff
@@ -1,31 +1,17 @@
 {
   "name": "<%= utils.sanitize(appname) %>",
-  "main": "src/main.ts",
+  "main": "./src/main.ts",
   "license": "SEE LICENSE IN <your-license-filename>",
   "version": "0.0.0",
   "scripts": {
     "postinstall": "node ./tools/postinstall.js && npm run ngcc",
-    "ngcc": "ngcc --tsconfig tsconfig.app.json --properties es2015 module main --first-only"
+    "ngcc": "<%= pathOffset %>node_modules/.bin/ngcc --tsconfig tsconfig.app.json --properties es2015 module main --first-only"
```

Comment:
@NathanWalker Many thanks for this update, works like a charm. Please be aware that the generated `../../node_modules/.bin/ngcc --tsconfig tsconfig.app.json --properties es2015 module main --first-only` will only work on unix machines. We have some windows developers in our team, they reported that this leads to an "Command '..' not found" error.

In our case this is not a problem since we're calling ngcc after install in the root package.json (which is in there by default, right?), so we deleted this call. 

Before I open an issue (for the windows users):
What's the reason you put this line here? Is it a "just to be sure" matter or is there any reason behind that I am missing?

Reply:
Thanks for bringing this up, I'm curious if you have a suggestion that would work well for windows here?
Essentially when running builds in CI in particular the postinstall does not pick up `ngcc` when referenced just like:
```
"ngcc": "ngcc --tsconfig tsconfig.app.json --properties es2015 module main --first-only"
```
Because `ngcc` is actually installed in root node_modules so the relative path ensures that it executes properly in those environments, however if Windows has an issue with that we need a way to execute `ngcc` properly. We had found during testing in some cases (did not seem 100%) however when `ngcc` did not process properly after a full clean then the first run would be subject to a runtime ivy related type of error (2nd run would be fine). So executing that after a clean seemed to always help.

Reply:
We can probably include that handling in the tools/postinstall.js itself and use something like this which would probably help both environments:
```
const child = childProcess.spawn(/^win/.test(process.platform) ? '..\\..\\node_modules\\.bin\\ngcc' : '../../node_modules/.bin/ngcc', ['--tsconfig', 'tsconfig.app.json', '--properties', 'es2015', 'module', 'main', '--first-only'], {
  cwd: process.cwd(),
  stdio: 'inherit',
});
child.on('close', (code) => {

});
```

Reply:
Great idea, moving to postinstall.js sounds good. Shall I create another PR?

Reply:
I have it here already, I'll publish another patch shortly with it ❤️  thanks again for mentioning!

Reply:
Awesome! Blazing fast, many thanks!!!

Reply:
Released in 2.0.2 now - changeset is here:
https://github.com/NativeScript/nx/commit/32284d7dd1df986e34c43e99eba9b78d27f3df7a